docs/cimpl6ToolingReference.md

```diff
@@ -0,0 +1,692 @@
+
+### Outputs from SHR-CLI
+
+The output of running SHR-CLI appear in a directory named "out". By default, the /out directory is created in the tooling directory. A different location can be designated using the `-o` [command line option](#executing-shr-cli).
+
+The content of the /out directory depends on which exporters were selected to run, which is controlled using the `-s` command line option. By default, the /out will contain five exports, as shown below, each corresponding to a separate export process:
+
+#### _Typical Contents of the /out Directory_
+
+![Typical Contents of the /out Directory](img_cimpl/out-directory.png)
+
+* cimcore - this directory is used in the process of building the "modeldoc" export, and is not discussed further
+* [data-dictionary](#data-dictionary-export) - this directory contains an MS-Excel spreadsheet containing a list of model elements and value sets
+* [fhir](#fhir-export) - this directory contains all the definitions and assets necessary to produce the IG
+* [json-schema](#json-schema-export) - this directory contains schemas for the (**TO DO -- need to define the JSON schema export**)
+* [modeldoc](#model-documentation-export) - this directory contains files that present the model with a look and feel similar to [Javadoc](#https://www.oreilly.com/library/view/learn-to-program/9781680500523/f_0126.html)
+
+## FHIR Export
+
+The FHIR export consists of profiles, extensions, value sets, code systems, examples, and other artifacts needed to create a FHIR IG. These are not the _pages_ of the IG, but rather various definitions that underlie the IG presentation. You must run the [IG publisher](#creating-the-implementation-guide) to create the IG itself.
+
+The FHIR export appears in the /fhir subdirectory under the designated "out" directory. The directory contains further subdirectories for code systems, extensions, the IG (guide), the logical model (logical), profiles, and value sets.
+
+### Logical Model Export
+
+The logical model is an alternative representation of the CIMPL model, expressed in a familiar FHIR-like format. The logical model closely follows the CIMPL definitions because it reflects the model _prior to_ mapping to FHIR resources. Each entry is presented as if it were a new FHIR resource, rather than a profile. The logical model is formally expressed as a set of FHIR `StructureDefinitions`. These definitions are distinguished from profiles because the names generated by the tooling end in `-model`.
+
+Because the logical model does not entail the additional complication of mapping, it is useful for review purposes. The logical model can be checked for clinical relevance and sufficiency before making mapping decisions.
+
+An optional part of the FHIR export, the logical model can be turned on or off using the `includeLogicalModels` flag in the configuration file.
+
+#### _Logical Model Example_
+![Logical Model Example](img_cimpl/logical-model-example.png)
```

Comment:
Diagram does not appear on MacOS systems. I suspect because of case sensitivity not detected in Windows - the actual file in the project is called `logical-model-example.PNG` where PNG was capitalized.  change this file as well as `model-doc-example.PNG` to lower case and it should resolve.

Reply:
good catch. resolved.